src/osgWrappers/serializers/osg/HeightField.cpp

```diff
@@ -31,18 +31,22 @@ static bool checkHeights( const osg::HeightField& shape )
 
 static bool readHeights( osgDB::InputStream& is, osg::HeightField& shape )
 {
-    osg::ref_ptr<osg::Array> array = is.readArray();
```

Comment:
lets change the FloatArray to be called farray to match OSG 3.6

Reply:
Fixed